debian/install

```diff
@@ -1,10 +1,10 @@
 accountsservice/io.elementary.pantheon.AccountsService.xml /usr/share/dbus-1/interfaces/
 accountsservice/io.elementary.pantheon.AccountsService.policy /usr/share/polkit-1/actions/
-appcenter.blacklist /etc/io.elementary.appcenter/
+appcenter/appcenter.blacklist /etc/io.elementary.appcenter/
 plank/dock1 /etc/skel/.config/plank/
 profile.d/*.sh /etc/profile.d/
-sessioninstaller.desktop /usr/share/applications/
-settings.ini /etc/gtk-3.0/
+sessioninstaller/sessioninstaller.desktop /usr/share/applications/
+gtk/settings.ini /etc/gtk-3.0/
 sudoers.d/* /etc/sudoers.d/
 overrides/*.gschema.override /usr/share/glib-2.0/schemas
-.inputrc /etc/skel/
+inputrc/.inputrc /etc/skel/
```

Comment:
shouldn't this file be completely unneeded https://www.debian.org/doc/manuals/maint-guide/dother.en.html#install

> If there are files that need to be installed into your package but your standard make install won't do it, put the filenames and destinations into this install file. They are installed by dh_install(1).

Reply:
It's necessary because we're generating multiple packages so we need to declare which package should contain which files

Reply:
Ahh, I guess that makes sense then.